sql/core/src/main/scala/org/apache/spark/sql/execution/exchange/EnsureRequirements.scala

```diff
@@ -55,6 +55,10 @@ case class EnsureRequirements(conf: SQLConf) extends Rule[SparkPlan] {
         child
       case (child, BroadcastDistribution(mode)) =>
         BroadcastExchangeExec(mode, child)
+      case (ShuffleExchangeExec(partitioning: RoundRobinPartitioning, child, _),
```

Comment:
How about use `Partitioning` instead of `RoundRobinPartitioning `. Since we already support this `SELECT /*+ REPARTITION(5, a) */ * FROM test ORDER BY a`.

Reply:
thanks, I will change it